tests/unittests/config/test_cc_apk_configure.py

```diff
@@ -60,6 +60,7 @@ def setUp(self):
         self.name = "apk_configure"
         self.cloud = cloud.Cloud(None, self.paths, None, None, None)
         self.args = []
+        temp_utils._TMPDIR = self.new_root
```

Comment:
This sets a global for that module. Shouldn't we clean this up after tests run?

Reply:
Should be fixed now